test/Microsoft.AspNetCore.Razor.Evolution.Test/Legacy/CSharpAutoCompleteTest.cs

```diff
@@ -121,7 +121,7 @@ public void VerbatimBlockAutoCompleteAtStartOfFile()
                                         Factory.Markup("<p>").Accepts(AcceptedCharacters.None)),
                                     new MarkupTagBlock(
                                         Factory.Markup("</p>").Accepts(AcceptedCharacters.None))),
-                               Factory.Span(SpanKind.Code, new CSharpSymbol(Factory.LocationTracker.CurrentLocation, string.Empty, CSharpSymbolType.Unknown))
```

Comment:
The location is now tracked by default. You don't need to pass it around as much